spec/haproxy.spec

```diff
@@ -124,6 +124,9 @@ fi
 %exclude %{_sbindir}/haproxy-systemd-wrapper
 
 %changelog
+* _COMPILE_DATE Mr Jenkins <jenkins@cakesolutions.net> - _VERSION
+- Update to _VERSION
+
```

Comment:
This won't keep the history unless you're re-committing this back to GitHub? Maybe just get rid of the rest of the ChangeLog and keep just your new templated entry?